### PR TITLE
Fix logging using arguments instead of formatter

### PR DESCRIPTION
**Pull request by vallsv**
Hi. While i was working around setuptools, here is very small clean up about code quality.
In case you take care of it.
Regards

### CODE REVIEW

File: setuptools/package_index.py [hunk 1]
```diff
@@ -1038,8 +1038,7 @@ def open_with_auth(url, opener=urllib.request.urlopen):
         cred = PyPIConfig().find_credential(url)
         if cred:
             auth = str(cred)
-            info = cred.username, url
-            log.info('Authenticating as %s for %s (from .pypirc)' % info)
+            log.info('Authenticating as %s for %s (from .pypirc)', cred.username, url)
```
Comment (jaraco) [inline setuptools/package_index.py]: I think this line is now too long.

**Comment by jaraco** [comment]
The rest of the changes are great, so I'll take the good with the bad. Thanks.

**Comment by vallsv** [comment]
you can use `*info` if you prefer. I can send a patch.

**Comment by vallsv** [comment]
Oh it is already done :-D
Thanks for the merge.

**Pull request merged**